src/features/query-bar/components/filter/Condition.js

```diff
@@ -109,22 +110,43 @@ const Condition = ({
         return [];
     };
 
+    const validateValue = (values: Array<ConditionValueType>, type: string) => {
+        switch (type) {
+            case NUMBER:
+                return !isInt(String(values[0]));
```

Comment:
Isn't this still returning `true` if invalid and `false` if valid?

Reply:
I think the naming/logic got kind of confusing. Here's the new setup:

```
const validateValue = (values: Array<ConditionValueType>, type: string) => {
        switch (type) {
            case NUMBER:
                return isInt(String(values[0]));
            case FLOAT:
                return isFloat(String(values[0]));
            default:
                break;
        }
        return true;
    };
```

validateValue will return true if the value is valid. It will return false if the value is invalid. isInt will return true if the string is an integer.

In getErrorMessage, I will have the variable be defined as `'isValueValid'` and have that be assigned to `validateValue(values, type)`.